release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1305,6 +1305,21 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-15-management-console-bug-fixes"]
 ==== Management Console
 
+* Previously, the *Expand PVC* modal assumed the existing PVC had a `spec.resources.requests.storage` value that includes a unit. As a result, when the *Expand PVC* modal was used to expand a PVC that had a `requests.storage` value without a unit, the console would display an incorrect value in the modal. With this update, the console was updated to handle storage values with and without a unit. (link:https://issues.redhat.com/browse/OCPBUGS-27909[*OCPBUGS-27909*])
+
+* Previously, the *Expand PVC* modal assumed the existing PVC had a `spec.resources.requests.storage` value that includes a unit. As a result, when the *Expand PVC* modal was used to expand a PVC that had a `requests.storage` value without a unit, the console would display an incorrect value in the modal. With this update, the console was updated to handle storage values with and without a unit. (link:https://issues.redhat.com/browse/OCPBUGS-27177[*OCPBUGS-27177*])
```

Comment:
Question: The description for this is the same as OCPBUGS-27909, but OCBUGS-27177 talks about cloning a PVC modal instead of expanding it, so does this description need to be about how the issue is resolved when cloning a PVC modal?

And if not, and the description is to be the same as OCPBUGS-27909, should they be combined into one entry?

Reply:
Removed the 27177 bug after consulting Olivia Payne in Slack.